modin/backends/pandas/query_compiler.py

```diff
@@ -1335,15 +1335,6 @@ def stack(self, level, dropna):
         )
     )
 
-    def repeat(self, repeats):
```

Comment:
I'm not sure why we are removing the implementation for `repeat`. Is it because we no longer support the `validate_index` argument for `map`?

Reply:
Yes, `repeat` was using `map` but it can change the number of rows (i.e. `explode`).